index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header class="toolbar_container">
+			<a href="#" class="Title_logo">My Logo</a>
+			<nav>
+				<img src="./images/i.png"  class="logo" alt="Icon menu">
+			</nav>
+		</header>
+
+		<main class="main_container">
+			<section class="text_container">
+				<h1 class="paragraphe">Hey There. <br/>I’m Leader.</h1>
+				
+				<h4>I'm a Software Developer</h4>
+				<p class="para">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+					don’t hesitate to contact me.</p>
+			</section>
+			
+			<section class="icons-container">
+				<ul class="icons-containers">
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon -GitHub.png"/></li>
+					<li><img src="./images/Icon - Linkedin.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Enabled.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Twitter.png"/></li>
+					<li><img src="./images/Normal Button/Tertiary/Icons/Icon - Medium.png"/></li>	
+				</ul>				
+			</section>
+		</main>
+	</section>
+	
+	<section class="second_container">
+		<div class="header_title">
+		<h2>My Recent Works</h2>
+		<p class="line"></p>
+		</div>
+		<div class="recent_work_container">
+			<img src="./images/imgs.png" alt="works_section">
+			<div class="recent_work_container_text">
+				<h3>Multi-Post Stories</h3>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+			<div class="butons_container">
+				<button type="button">CSS</button>
+				<button type="button">HTML</button>
+				<button type="button">Boostrap</button>
+				<button type="button">Ruby</button>
+			</div>
+			<button  type="button" class="project_btn">See project</button>
+			</div>
+		</div>
+	</section>
```

Comment:
- [ ] According to your previous reviewer kindly fix the font weight of the list of technologies
 the thickness of the divider (horizontal line)
 your button is not well aligned with the center

- [ ] your design
![Screenshot 2023-07-15 025022](https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/92458236/39158f13-af2f-4848-8f9b-50d2119d5b3a)

- [ ] Figma design 
![image](https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/92458236/e27f82e2-7f88-4059-a2b4-eeb07a7341f0)


- [See Previouse Review](https://github.com/malodev1234/Portfolio_setup_mobile_first/pull/3#discussion_r1264115349)